src/utils/LiftUtils.ts

```diff
@@ -0,0 +1,11 @@
+import { Set } from "../types/LiftTypes";
+
+export const allRepsAreEqual = (sets: Set[]) =>
+  sets.every(s => s.reps === sets[0].reps);
```

Comment:
I know it's my code, but how safe is `sets[0]`. It doesn't crash, but rather returns `undefined` in case of an empty array, doesn't it?
EDIT: `sets[0].reps` would fail on an empty array though?

Reply:
It will crash if the array is empty, will fix it.